srcpkgs/OpenCoarrays/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'OpenCoarrays'
+pkgname=OpenCoarrays
+version=2.9.2
+revision=1
+build_style=cmake
+short_desc="OpenCoarray fortran PAGS paralellization library"
+maintainer="Alberto Ramos <alberto.ramos@ific.uv.es>"
+hostdepends="gcc gcc-fortran openmpi openmpi-devel cmake"
+makedepends="gcc gcc-fortran openmpi openmpi-devel cmake"
+depends="gcc gcc-fortran openmpi openmpi-devel cmake"
+license="GPL-3.0-or-later"
```

Comment:
The license is actually a BSD 3 Clause license, change to `license="BSD-3-Clause"`, and add
```
post_install() {
    vlicense LICENSE
}
```
but with a tab before vlicense, not spaces.